central/processlisteningonport/store/postgres/full_store.go

```diff
@@ -270,3 +304,27 @@ func (s *fullStoreImpl) readRows(
 
 	return plops, nil
 }
+
+func (s *fullStoreImpl) readCountRows(
+	ctx context.Context,
+	rows pgx.Rows,
+) (map[string]int32, error) {
+	var counts = make(map[string]int32)
+
+	for rows.Next() {
+		var deploymentID string
+		var namespace string
+		var clusterID string
+		var count int32
+
+		if err := rows.Scan(&deploymentID, &namespace, &clusterID, &count); err != nil {
+			return nil, pgutils.ErrNilIfNoRows(err)
+		}
+
+		if ok, err := plopSAC.ReadAllowed(ctx, sac.ClusterScopeKey(clusterID), sac.NamespaceScopeKey(namespace)); err == nil && ok {
```

Comment:
Is the only reason we include cluster and namespace in this query is for SAC?  If so could we put in a comment regarding that?

Reply:
Yes. I have included a comment on sql query to explain that.